utils/describe-ref/lib/describe-ref.js

```diff
@@ -7,15 +7,16 @@ module.exports = describeRef;
 module.exports.parse = parse;
 module.exports.sync = sync;
 
-function getArgs(options) {
+function getArgs(options, lastRev) {
   const args = [
     "describe",
+    // fetch it from the latest tag, no matter in which branch
```

Comment:
That's not what the `--tags` option of `git describe` [does](https://git-scm.com/docs/git-describe#git-describe---tags). You're enabling "lightweight" tags to be consulted, which is _not_ correct.